db/people.yml

```diff
@@ -2636,8 +2636,20 @@ pnehra:
   active: true
   aka: []
   avatar: "https://github.com/priyankaanehra.png"
-  bio: bio
-  education: []
+  bio: Priyanka is from BC, Canada. She is currently an undergraduate student
+       studying applied mathematics in SEAS. For the past 3 years, she studied
+       mathematics with a pure math focus in Virginia, while also minoring in
+       physics and conducting research in the chemistry field. In her spare
+       time, she enjoys running and baking.
+  education:
+    - degree: BS in Applied Mathematics
+      institution: Columbia University
+      end_year: 2022
+      begin_year: 2020
+    - degree: BS in Mathematics
+      institution: University of Richmond
+      end_year: 2020
+      begin_year: 2017
   email: pn2365@columbia.edu
   employment:
     - begin_date: 2020-10-14
```

Comment:
this is not your fault but please can you change the end-date to 2021, that is a typo made by me I guess.....

Thanks